### ISSUE TITLE
Option to exclude directories

### ISSUE THREAD

**Issue by sparx82** [simon987/sist2#504]
**Which SIST2 component is your Feature Request related to?**
Scan

**Is your feature request related to a problem? Please describe.**
No

**What would you like to see happen?**
I have certain directories which contain a very large amount of files which don't need to be scanned and indexed. It would be nice to have an option to exlucde certain directores within the general base path.

**Additional context**
N/A

**Comment by dpieski** [comment]
You can already do that with the exclusions.
e.g. `(/foo1/bar)|(/foo2/bar)` will exclude the `foo1/bar` folder and the `foo2/bar` folder.

**Comment by sparx82** [comment]
Yes, I figured that out but I also have filetypes I whish to enclude and ended up with that:
`(^.*\.(cr2|CR2|cr3|CR3|dng|DNG|epub|zip|ZIP|tar.gz|TAR.GZ|calibz|7z|7Z|nef|NEF|jpg|JPG|png|PNG|jpeg|JPEG)$)|(\/path\/to\/excludeddir\/.*)`
which works but is not exactly convenient...

**Comment by shyyio** [comment]
See https://github.com/sist2app/sist2/blob/master/docs/USAGE.md#excluding-files
